src/main/java/edu/harvard/iq/dataverse/settings/JvmSettings.java

```diff
@@ -105,6 +105,10 @@ public enum JvmSettings {
     SCOPE_PID_HANDLENET_KEY(SCOPE_PID_HANDLENET, "key"),
     HANDLENET_KEY_PATH(SCOPE_PID_HANDLENET_KEY, "path", "dataverse.handlenet.admcredfile"),
     HANDLENET_KEY_PASSPHRASE(SCOPE_PID_HANDLENET_KEY, "passphrase", "dataverse.handlenet.admprivphrase"),
+
+    // EXPORT SETTINGS
+    SCOPE_EXPORTERS(PREFIX, "exporters"),
```

Comment:
May I suggest we add the scope "exporters" to a another scope "plugins"? The name now seems a bit arbitrary. Is it a target dir where exports are written? Adding "plugins" (or sth similar) make the intention clearer and will allow us to add more switches (like enable/disable, lists, options, ...)